pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[project]
+name = "joligen"
+version = "1.0.0"
+description = "Generative AI toolset with GANs and diffusion for real-world applications."
+readme = "README.md"
+keywords = [
+  "Diffusion Model",
+  "GAN",
+  "Generative AI",
+  "Image Generation",
+  "Image to Image Translation",
+  "Inpainting",
+  "PyTorch",
+  "Super Resolution",
+]
+license = { file = "LICENCE" }
+authors = [
+    { name = "JoliBrain", email = "contact@jolibrain.com" },
+]
+requires-python = ">=3.8"
+classifiers = [
+  "Operating System :: POSIX :: Linux",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+]
+dependencies = [
+  "addict",
+  "aim",
+  "dominate",
+  "einops",
+  "imgaug",
+  "numpy>=1.20.0",
+  "onnx",
+  "Pillow",
+  "piq",
+  "positional-encodings[pytorch]",
+  "pydantic",
+  "segment-anything@ git+https://github.com/facebookresearch/segment-anything.git",
+  "thop",
+  "timm",
+  "torch==1.13.1",
+  "torchinfo",
+  "torchvision==0.14.1",
+  "torchviz",
+  "tqdm",
+  "visdom",
+  "vision-aided-loss@ git+https://github.com/nupurkmr9/vision-aided-gan.git",
+  "wget",
+]
+[project.urls]
+Documentation = "https://joligen.readthedocs.io"
```

Comment:
The documentation won't be hosted on readthedocs (rather docs.joligen.com when v1.0 is published), atm it's on [https://joligen.com/beta/doc/](https://joligen.com/beta/doc/)

Reply:
> The documentation won't be hosted on readthedocs (rather docs.joligen.com when v1.0 is published), atm it's on https://joligen.com/beta/doc/

Now at https://joligen.com/doc/